sql/size_utils.sql

```diff
@@ -29,8 +29,8 @@ FROM
     _timescaledb_catalog.hypertable h
     JOIN _timescaledb_catalog.chunk c ON h.id = c.hypertable_id
         AND c.dropped IS FALSE
-    JOIN LATERAL _timescaledb_internal.relation_size(
-        format('%I.%I'::text, c.schema_name, c.table_name)::regclass) AS relsize ON TRUE
+    JOIN pg_class cl ON cl.relname = c.table_name AND cl.relkind = 'r'
```

Comment:
Here you also should join with pg_namespace catalog table

Reply:
Done.